server/testutils/factories/booking.ts

```diff
@@ -86,8 +87,8 @@ class BookingFactory extends Factory<Booking> {
 
 export default BookingFactory.define(() => ({
   person: personFactory.build(),
-  arrivalDate: faker.date.soon().toISOString(),
-  departureDate: faker.date.future().toISOString(),
+  arrivalDate: formatISO(faker.date.soon(), { representation: 'date' }),
+  departureDate: formatISO(faker.date.future(), { representation: 'date' }),
```

Comment:
I'm happy to implement it but shouldn't we use the same date format across all our factories?

Reply:
Yeah, I kinda want to pick these things up as we encounter them, but I think we'll need to.